dev/devicelab/bin/tasks/flutter_run_test.dart

```diff
@@ -31,35 +33,47 @@ Future<TaskResult> createFlutterRunTask() async {
   bool failedTest = false;
   bool skippedTest = false;
   bool finishedMessage = false;
+  bool printMessage = false;
+  bool writelnMessage = false;
   final Device device = await devices.workingDevice;
   await device.unlock();
   final List<String> options = <String>[
-    '-t', runTestSource.absolute.path, '-d', device.deviceId,
+    '-t', runTestSource.absolute.path, '-d', device.deviceId, '-v',
   ];
   await inDirectory<void>(flutterGalleryDir, () async {
-    startProcess(
+    final Process run = await startProcess(
       path.join(flutterDirectory.path, 'bin', 'flutter'),
       flutterCommandArgs('run', options),
       environment: null,
     );
+
     final Completer<void> finished = Completer<void>();
-    final StreamSubscription<void> subscription = device.logcat.listen((String line) {
+    final StreamSubscription<void> subscription = run.stdout
+        .transform<String>(utf8.decoder)
+        .transform<String>(const LineSplitter())
+        .listen((String line) {
+      print('stdout: $line');
       // tests execute in order.
       if (line.contains(passedMessageMatch)) {
         passedTest = true;
       } else if (line.contains(failedMessageMatch)) {
         failedTest = true;
       } else if (line.contains(skippedMessageMatch)) {
         skippedTest = true;
+      } else if (line.contains(printMessageMatch)) {
+        printMessage = true;
+      } else if (line.contains(writelnMessageMatch)) {
+        writelnMessage = true;
       } else if (line.contains(finishedMessageMatch)) {
         finishedMessage = true;
         finished.complete();
       }
     });
     await finished.future.timeout(const Duration(minutes: 1));
     subscription.cancel();
+    run.kill();
   });
-  return passedTest && failedTest && skippedTest && finishedMessage
+  return passedTest && failedTest && skippedTest && finishedMessage && printMessage && writelnMessage
```

Comment:
This fails on stable (neither line is printed).

Reply:
👏